src/python/pants/help/help_info_extracter.py

```diff
@@ -236,6 +236,9 @@ def get_all_help_info(
         scope_to_help_info = {}
         name_to_goal_info = {}
         for scope_info in sorted(options.known_scope_to_info.values(), key=lambda x: x.scope):
+            if scope_info.scope.startswith("_"):
+                # Exclude "private" subsystems.
+                continue
```

Comment:
This is to hide the `__no_goal` and `__unknown_goal` built in goals from help output.